neurom/core/_soma.py

```diff
@@ -75,15 +81,15 @@ class SomaSinglePoint(Soma):
     Represented by a single point.
     """
 
-    def __init__(self, points):
+    def __init__(self, morphio_soma):
         """Initialize a SomaSinglePoint object."""
-        super().__init__(points)
-        self.radius = points[0][COLS.R]
+        super(SomaSinglePoint, self).__init__(morphio_soma)
+        self.radius = self.points[0][COLS.R]
 
     def __str__(self):
         """Return a string representation."""
         return ('SomaSinglePoint(%s) <center: %s, radius: %s>' %
-                (repr(self._points), self.center, self.radius))
+                (repr(self.points), self.center, self.radius))
 
 
 class SomaCylinders(Soma):
```

Comment:
Its docstring uses `points`. This needs to be updated to `morphio_soma`

Reply:
why did you have to change points to morphio_soma?

Reply:
understood, it is not an array of points, but a morphio object ok!